### PR TITLE
[ADD] fieldservice_crm

### PR DESCRIPTION
**Pull request by patrickrwilson**
This adds a new module called fieldservice_crm which connects field service orders with the sales crm module. This is useful for when personnel are needed to travel to the location for evaluation or surveys before a quote gets generated.

### CODE REVIEW

**Comment by patrickrwilson** [comment]
ping @max3903 and @osi-scampbell

**Review by brian10048 — CHANGES_REQUESTED** [review]
Great addition!

Can you add an option to enable this module in the base field service settings?

**Comment by patrickrwilson** [comment]
> Great addition!
> 
> Can you add an option to enable this module in the base field service settings?

Yes, I won't be able to get to this until maybe later next week but will definitely add that.

**Comment by patrickrwilson** [comment]
@brian10048 updated the settings to include this module so I think it's ready for more review. 
Ping @max3903 @osi-scampbell

File: fieldservice_crm/views/fsm_location.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        <field name="inherit_id" ref="fieldservice.fsm_location_form_view"/>
+        <field name="arch" type="xml">
+            <div class="oe_button_box" position="inside">
+                <button class="oe_stat_button o_res_partner_tip_opp" type="action"
+                        name="%(crm.crm_lead_opportunities)d"
+                        icon="fa-star"
+                        groups="sales_team.group_sale_salesman"
+                        context="{'search_default_fsm_location_id': active_id,
+                                  'default_fsm_location_id': active_id}">
+                    <field string="Opportunites" name="opportunity_count" widget="statinfo"/>
```
Comment (max3903) [inline fieldservice_crm/views/fsm_location.xml]:
```suggestion
                    <field string="Opportunities" name="opportunity_count" widget="statinfo"/>
```

**Review by max3903 — APPROVED** [review]

**Review by brian10048 — APPROVED** [review]

**Pull request merged**